developer/src/kmc-model/src/build-trie.ts

```diff
@@ -291,13 +291,28 @@ namespace Trie {
    * @returns A JSON-serialiable object that can be given to the TrieModel constructor.
    */
   export function buildTrie(wordlist: WordList, keyFunction: SearchTermToKey): object {
-    let root = new Trie(keyFunction).buildFromWordList(wordlist).root;
+    let trie = new Trie(keyFunction);
+    buildFromWordList(trie, wordlist);
+    const root = trie.root;
     return {
       totalWeight: sumWeights(root),
       root: root
     }
   }
 
+  /**
+   * Populates the trie with the contents of an entire wordlist.
+   * @param words a list of word and count pairs.
+   */
+  function buildFromWordList(trie: Trie, words: WordList): Trie {
```

Comment:
Given we now pass `trie` in as a parameter here, and we aren't using the return value on line 295, we should probably make this a `void` type.

Also, please add `@param` doc for `trie`.